target/linux/ramips/image/mt7688.mk

```diff
@@ -34,3 +34,21 @@ define Device/widora-neo
   DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci
 endef
 TARGET_DEVICES += widora-neo
+
+define Device/Omega2
+  DTS := OMEGA2
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  SUPPORTED_DEVICES := omega2
+  DEVICE_TITLE := Onion Omega2
+  DEVICE_PACKAGES:= kmod-usb2 kmod-usb-ohci uboot-envtools
+endef
+TARGET_DEVICES += Omega2
+
+define Device/Omega2P
+  DTS := OMEGA2P
+  IMAGE_SIZE := $(ralink_default_fw_size_32M)
+  SUPPORTED_DEVICES := omega2p
+  DEVICE_TITLE := Onion Omega2+
+  DEVICE_PACKAGES:= kmod-usb2 kmod-usb-ohci uboot-envtools
```

Comment:
I would add kmod-sdhci-mt7620 in the DEVICE_PACKAGES, because the Omega2+ has a microSD-slot and it makes sense to have it actually working.